src/imitation/algorithms/adversarial.py

```diff
@@ -36,15 +35,15 @@ class AdversarialTrainer:
     def __init__(
```

Comment:
Nit: docstring should be updated to say "Builds AdversarialTrainer" not "Builds Trainer".